acra-censor/handlers/handlers_util_test.go

```diff
@@ -536,5 +536,287 @@ func TestPatternsInWhereClauses(t *testing.T) {
 			}
 		}
 	}
+}
+
+func TestOrderByWithColumnPattern(t *testing.T) {
+	patterns := []string{
+		// ORDER BY with column placeholder
+		fmt.Sprintf("SELECT a1 FROM table1 ORDER BY %s", ColumnConfigPlaceholder),
```

Comment:
please add one more pattern:
select a from b order by a, %%column%%, 1, %%column%%

match queries:
1. select a from b order by a, 2, 1, ABC

not match:
1. select a from b order by 1, abc, 1, abs
2. select a from b order by a, abc, a, ABC
3. select a from b order by a, abc, 1, ABC, ABC

I think it will not pass now because in processing order by we doesn't use reflect DeepEqual after pattern mismatch in a loop

Reply:
Agree. Thanks. It seems that the same it true for GroupBy